examples/3d_examples/OctopusArmCase3D/run_tapered_arm_and_sphere_with_flow.py

```diff
@@ -242,10 +242,11 @@ def tapered_arm_and_cylinder_flow_coupling(
                     h5_file_name="sopht_" + str("%0.4d" % (time * 100)) + ".h5",
                     time=time,
                 )
-                # Need to first update element position of rod since this is not updated
-                # when timestepping in elastica (since it is not needed).
-                cosserat_rod_flow_interactor.update_rod_element_position()
-                rod_io.save(
+                forcing_grid_io.save(
+                    h5_file_name="forcing_grid_" + str("%0.4d" % (time * 100)) + ".h5",
+                    time=time,
+                )
```

Comment:
see above.

Reply:
Fixed in [722f451](https://github.com/SophT-Team/SophT-Simulator/pull/70/commits/722f451c2f4e0b8a43da93ede8af488d35a7eba5)